public/checkout.js

```diff
@@ -457,6 +457,7 @@ var packeteryLoadCheckout = function( $, settings ) {
 				widgetOptions.layout = 'cd';
 				widgetOptions.appIdentity = settings.appIdentity;
 				widgetOptions.expeditionDay = settings.expeditionDay;
+				widgetOptions.sample = !!settings.isCarDeliverySampleEnabled;
```

Comment:
Nelíbí se mi použití dvojité negace pro získání booleanu. Když tam bude string "false", tak to bude true. Preferuji zajistit správnou hodnotu již při předání dat do settings.

Reply:
S timhle si dovolim nesouhlasit. Kdyz tam das string "false", tak ti to rupne pri validai, ze jsi tomu nedal boolean. Ovsem jinym zpusobem jsem to tez dokazal udelat, ale to si nemyslim, ze vypada o mec lepe, alespon me se to tak zda. Misto negace se da pouzit `settings.isCarDeliverySampleEnabled == 1;`, ale nic jineho me zatim nenapadlo. Mas nejaky konkretni postup na mysli?

Reply:
settings.isCarDeliverySampleEnabled je string, buď '' nebo '1', takže se dá použít settings.isCarDeliverySampleEnabled === '1' nebo to vyřešit jako s widgetAutoOpen

Reply:
Zvolil jsem tu druhou moznost.